avocado/utils/software_manager.py

```diff
@@ -517,6 +536,72 @@ def provides(self, name):
         else:
             return None
 
+    def build_dep(self, name):
+        """
+        Install build-dependencies for package [name]
+
+        :param name: name of the package
+
+        :return True: If build dependencies are installed properly
+        """
+        option = '-y --tolerant'
+        cmd = 'yum-builddep %s %s' % (option, name)
```

Comment:
do you have any reasons not to use directly `cmd = "yum-builddep -y --tolerant %s" % name`? ... or even `process.system("yum-builddep -y --tolerant %s" % name, sudo=True)`?

Reply:
no, will change it accordingly.